syntax = "proto3";

message CurrentLocation {
    double latitude = 1;
    double longitude = 2;
}

message Device {
    string deviceId = 1;
    bool isActive = 2;
    uint32 batteryLevel = 3;
    CurrentLocation currentLocation = 4;
    string lastErrorMessage = 5;
    repeated string allowedOperations = 6;
}
